descwl_shear_sims/simple_sim.py

```diff
@@ -246,7 +249,12 @@ def __init__(
         self._final_gal_kws = gal_kws_defaults
 
         self.psf_type = psf_type
-        self.psf_kws = psf_kws or {'fwhm': 0.8}
+        self.psf_kws = copy.deepcopy(PSF_KWS_DEFAULTS[self.psf_type])
+        if psf_kws is not None:
+            self.psf_kws.update(psf_kws)
+        self.psf_g1 = self.psf_kws.pop('psf_g1', 0.0)
+        self.psf_g2 = self.psf_kws.pop('psf_g2', 0.0)
```

Comment:
Why does it say `psf_g1` if it is in the psf_kws in the first place? We can use `g1` only I think.

Reply:
good point